docs/courseInstance/index.md

```diff
@@ -154,11 +154,91 @@ To migrate from `allowAccess` to publishing extensions:
 
 ## Enrollment controls
 
-Students can enroll in a course instance through one of two ways:
+!!! warning "Not fully available yet"
+
+    The new self-enrollment system will be released in January 2026. Currently, your course instance will always be available for self-enrollment via a direct link, you cannot block users, or limit self-enrollment to specific institutions. The only configurable setting until the new system is fully available is the `hideInEnrollPage` setting.
+
+    The default values for the new system:
+
+    ```json title="infoCourseInstance.json"
+    {
+      "hideInEnrollPage": /* current value of hideInEnrollPage */,
+      "selfEnrollment": { /* None of these settings are modifiable in the old system */
+        "enabled": true,
+        "restrictToInstitution": false,
+        "beforeDate": null,
+        "useEnrollmentCode": false,
+      }
+    }
+    ```
+
+Students can enroll in a course instance through a few different ways:
 
 1. They can use a URL specific to the course instance or [to one of its assessments](../assessment/configuration.md#linking-to-assessments). You can find the "student link" on the "Settings" tab of the course instance. This link points students to the list of assessments associated to the course instance, enrolling them automatically in the course instance if they are not yet enrolled.
 
-2. They can use the "Add or remove courses" button on PrairieLearn's homepage. This button opens a page listing all course instances that are currently available for enrollment, giving students the option to add new courses.
+2. They can use a self-enrollment code. You can find the enrollment code on the "Settings" tab of the course instance after enabling "Use enrollment code for self-enrollment". Students can enroll in a course via the PrairieLearn homepage and input the enrollment code.
```

Comment:
Could we say which button the student should click on the homepage? I'm thinking about "Students can enroll in a course by clicking on the XXXXXX button on their PrairieLearn homepage and entering the enrollment code."

Reply:
Changed in [a7b4901](https://github.com/PrairieLearn/PrairieLearn/pull/13437/commits/a7b4901e9b37b30a011664d0612113cc5de25766).